dstep/driver/Application.d

```diff
@@ -37,20 +38,24 @@ class Application : DStack.Application
 
     private
     {
-        Language language;
+        Options defaultOptions;
```

Comment:
Well, it is initialized in methods: `handleArguments` and `handleObjectiveC`, so it cannot be enum.